FilterViewController.h

```diff
@@ -12,5 +12,6 @@
 @interface FilterViewController : UITableViewController<MHCollapibleViewManagerDelegate>
 
 @property (strong, nonatomic) NSMutableArray *managerArray;
+@property IBOutlet UIButton *clearButton;
```

Comment:
IBOutlets almost never need to be public. If this doesn't need to be used outside of this class could you move it's definition to the .m file.